designs/Verify-Verbosity.md

```diff
@@ -0,0 +1,240 @@
+```
+
+### Log level mapping - The following details are copied from [here](https://github.com/NuGet/Home/blob/dev/designs/Package-List-Verbosity.md#log-level-mapping). Thanks to [Joel Verhagen](https://github.com/joelverhagen) for the detailed information.
+
+The provided `--verbosity` value will include to NuGet log messages with the following levels:
+
+​             | `q[uiet]` | `m[inimal]` | `n[ormal]` | `d[etails]` | `diag[nostic]`
+------------- | --------- | ----------- | ---------- | ----------- | --------------
+`Error`       | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+`Warning`     | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+`Minimal`     | ❌        | ✔️         | ✔️         | ✔️         | ✔️   
+`Information` | ❌        | ❌         | ✔️         | ✔️         | ✔️   
+`Verbose`     | ❌        | ❌         | ❌         | ✔️         | ✔️   
+`Debug`       | ❌        | ❌         | ❌         | ✔️         | ✔️   
+
+Note that MSBuild itself has the following mapping for it's own "log levels"
+([source](https://docs.microsoft.com/en-us/visualstudio/msbuild/obtaining-build-logs-with-msbuild?view=vs-2019#verbosity-settings)):
+
+​                                     | `q[uiet]` | `m[inimal]` | `n[ormal]` | `d[etailed]` | `diag[nostic]`
+------------------------------------- | --------- | ----------- | ---------- | ----------- | --------------
+Errors                                | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+Warnings                              | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+High-importance Messages              | ❌        | ✔️         | ✔️         | ✔️         | ✔️   
+Normal-importance Messages            | ❌        | ❌         | ✔️         | ✔️         | ✔️   
+Low-importance Messages               | ❌        | ❌         | ❌         | ✔️         | ✔️   
+Additional MSBuild-engine information | ❌        | ❌         | ❌         | ❌         | ✔️   
+
+In other words, you can think of the following log concepts per row as equivalent.
+
+NuGet log level    | MSBuild verbosity switch | MSBuild message type
+------------------ | ------------------------ | --------------------
+`Error`, `Warning` | `q[uiet]`                | Errors and Warnings
+`Minimal`          | `m[inimal]`              | High-importance Messages
+`Information`      | `n[ormal]`               | Normal-importance Messages
+`Verbose`, `Debug` | `d[etailed]`             | Low-importance Messages
+​                   | `diag[nostic]`           | Additional MSBuild-engine information
+
+### Example output
+
+This is output of the invocation of `dotnet nuget verify nuget.common.5.9.0-preview.2.nupkg`
+
+#### Default verbosity | `--verbosity mimimal`
+
+```
+Verifying NuGet.Common.5.9.0-preview.2
+Signature type: Author
+Verifying the author primary signature with certificate:
+  SHA256 hash: 3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE
```

Comment:
Could we add subject names to the minimal output for human readability purposes?

Reply:
sure, we can do that but I followed @dtivel suggestion https://github.com/NuGet/NuGet.Client/pull/3774#issuecomment-740216665

Reply:
It's totally fine with me to add subject name, but it needs to be on every signature and in both tools.  Given that a package can have up to 4 total signatures (i.e.:  author, author timestamp, repository countersignature, and repository countersignature timestamp), this will increase output by up to 4 lines per signed package.  Subject name is more human readable than a fingerprint, so it seems like an acceptable tradeoff for slightly less minimalistic logging.

Reply:
Thanks, Damon, for the feedback. The following fields are `included` for every certificate in the `minimal verbosity`.
- `Subject Name`
- `SHA256 hash`
- "Validity`

The following fields are `excluded` for every certificate in the `minimal verbosity`.
- `SHA1 hash`
- `Issued by`